api/pacman-api-compliance/src/main/java/com/tmobile/pacman/api/compliance/repository/ComplianceRepositoryImpl.java

```diff
@@ -2342,9 +2344,13 @@ public IssueExceptionResponse exemptAndUpdateMultipleIssueDetails(String assetGr
                         String _index = dataSource;
                         String _type = targetType + "_audit";
                         if(!skipAuditTrail) {
-                            Map<String, String> optionalAuditFields = new HashMap<>(2);
-                            optionalAuditFields.put(Constants.ISSUE_EXEMPTION_EXPIRY_DATE, sdf.format(issuesException.getExceptionEndDate()));
-                            optionalAuditFields.put(Constants.ISSUE_EXEMPTION_REASON, issuesException.getExceptionReason());
+                            Map<String, String> optionalAuditFields = Collections.emptyMap();
+                            if (issuesException.getExceptionEndDate() != null) {
+                                optionalAuditFields.put(Constants.ISSUE_EXEMPTION_EXPIRY_DATE, sdf.format(issuesException.getExceptionEndDate()));
+                            }
+                            if (isNotBlank(issuesException.getExceptionReason())) {
+                                optionalAuditFields.put(Constants.ISSUE_EXEMPTION_REASON, issuesException.getExceptionReason());

```

Comment:
Same here, I believe reason and expiration date are mandatory fields on the UI. If they are, you might not be able to create an exemption.